testing/distributions-integ-tests/src/integTest/groovy/org/gradle/AllDistributionIntegrationSpec.groovy

```diff
@@ -37,8 +37,8 @@ class AllDistributionIntegrationSpec extends DistributionIntegrationSpec {
     }
 
     @Override
-    int getMaxDistributionSizeBytes() {
-        return 226 * 1024 * 1024
+    int getDistributionSizeMb() {
```

Comment:
Can you use `MiB` (mebibyte) instead of `Mb` (megabit).  `Mb` implies this is (212*1000*1000/8) bytes.